sentinel-extension/sentinel-metric-exporter/src/main/java/com/alibaba/csp/sentinel/metric/MetricExporterInit.java

```diff
@@ -0,0 +1,81 @@
+/*
+ *  Copyright 1999-2021 Alibaba Group Holding Ltd.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+package com.alibaba.csp.sentinel.metric;
+
+import com.alibaba.csp.sentinel.init.InitFunc;
+import com.alibaba.csp.sentinel.log.RecordLog;
+import com.alibaba.csp.sentinel.metric.collector.MetricCollector;
+import com.alibaba.csp.sentinel.metric.exporter.MetricExporter;
+import com.alibaba.csp.sentinel.metric.exporter.jmx.JMXMetricExporter;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * The{@link MetricExporterInit} work on load Metric exporters.
+ *
+ * @author chenglu
+ * @date 2021-07-01 19:58
+ * @since 1.8.3
+ */
+public class MetricExporterInit implements InitFunc {
+    
+    /**
+     * the list of metric exporters.
+     */
+    private static List<MetricExporter> metricExporters = new ArrayList<>();
+    
+    /**
+     * the metric collector.
+     */
+    private static MetricCollector metricCollector = new MetricCollector();
```

Comment:
I think it's not necessary to keep the reference here. And even it's not necessary to pass the reference into the constructor of JMXMetricExporter.

Reply:
Yes, Now it can be kept into `JMXMetricExporter` directly instead of though constructor.